lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -361,14 +357,9 @@ def handle_send_image_mode(self, event):
         # The client requests notification of what the current image mode is.
         self.send_event('image_mode', mode=self._current_image_mode)
 
-    def handle_set_dpi_ratio(self, event):
-        dpi_ratio = event.get('dpi_ratio', 1)
-        if dpi_ratio != self._dpi_ratio:
-            # We don't want to scale up the figure dpi more than once.
-            if not hasattr(self.figure, '_original_dpi'):
-                self.figure._original_dpi = self.figure.dpi
-            self.figure.dpi = dpi_ratio * self.figure._original_dpi
-            self._dpi_ratio = dpi_ratio
+    def handle_set_device_pixel_ratio(self, event):
```

Comment:
I guess these handlers are effectively private and we reserve the right to change them as we see fit?

Reply:
That's a good point, but I would say so. They're tied directly with the JavaScript side of the implementation, and I don't think we've ever guaranteed anything there.

In this case, it's only a rename for consistency's sake, so I could revert it.

Reply:
This will likely require a change in ipympl, but I think we should make the changes to keep the function names consistent (in this case).

Reply:
I restored the handler for `set_dpi_ratio`, as then one can continue using an older version of `ipympl` with the new Matplotlib that comes out with this change.